dashboard/views.py

```diff
@@ -6,7 +7,19 @@
 from push.models import PushApplication
 
 
-class Home(TemplateView):
+class Login(TemplateView):
+    template_name = 'dashboard/login.html'
+
+
+class Status(TemplateView):
+    template_name = 'dashboard/status.html'
+
+
+class PermissionDenied(TemplateView):
+    template_name = 'dashboard/errors/403.html'
+
+
+class Home(LoginRequiredMixin, TemplateView):
```

Comment:
Yeah, it only makes sense if everything on the homepage is behind login. If we change the homepage this won't make sense.